app/models/concerns/lockable.rb

```diff
@@ -4,7 +4,7 @@ module Lockable
   extend ActiveSupport::Concern
 
   included do
-    has_one :lock, as: :resource, dependent: :destroy
+    has_many :lock, as: :resource, dependent: :destroy
```

Comment:
this should be locks and lock should be marked deprecated but kept around for at least the api